client/benches/tps/lib.rs

```diff
@@ -75,10 +81,12 @@ impl Config {
         for handle in handles {
             handle.join().expect("Event counter panicked")?;
         }
+
         // END
         let elapsed_secs = timer.elapsed().as_secs_f64();
-        thread::sleep(core::time::Duration::from_secs(2));
-        let blocks_out_of_measure = 1 + 2 * self.peers;
+        // Sleep to let the blocks to be committed on all peers
+        thread::sleep(core::time::Duration::from_secs(5));
```

Comment:
This _should_ not be needed because the handles joined means all block-committed events has been detected, but I remember I observed what contrary to my expectation

Reply:
Yeah, but sometimes this bench is still failing with message about block not being committed yet...

Reply:
Not _committed_ but _applied_. Before this bench was broken, only few sleep between the commit and the apply had been needed.
 
Can be left to #2225. See https://github.com/hyperledger/iroha/issues/2225#issuecomment-1144787908.

Reply:
@s8sato , could you also approve #2489 ?